spec/rubocop/cop/lint/object_equality_override_spec.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Lint::ObjectEqualityOverride, :config do
+  let(:msg) { 'Check against argument class within custom equality methods.' }
+
+  it 'does not register an offense when classes are checked with `==` method' do
+    expect_no_offenses(<<~RUBY)
+      def eql?(other)
+        self.class == other.class && self.title == other.title
+      end
+    RUBY
+  end
+
+  it 'does not register an offense when classes are checked with `<=` method' do
+    expect_no_offenses(<<~RUBY)
+      def eql?(other)
+        other.class <= self.class && self.title == other.title
+      end
+    RUBY
+  end
```

Comment:
Shouldn't the same thing apply for the `<` operator?

Reply:
Yep, that's true. Fixed.